recipe/build.sh

```diff
@@ -61,16 +68,16 @@ fi
 
 # SEE PR #5 (can't build to do aligned_alloc missing on osx)
 if [[ $(uname) == "Darwin" ]]; then
-	USE_LIBXSMM=OFF
+    USE_LIBXSMM=OFF
     USE_LIBURING=OFF
```

Comment:
By the way, if you want to switch this on on osx, we only need to bump `c_stdlib_version` to 10.15